coordinator/txmanager.go

```diff
@@ -161,7 +157,7 @@ func (t *TxManager) NewAuth(ctx context.Context, batchInfo *BatchInfo) (*bind.Tr
 	}
 
 	maxGasPriceBig := big.NewInt(maxGasPrice)
```

Comment:
Not related to this PR, but I did not find where the maxGasPrice value from config file, also I checked with @tclemos and @jeffprestes and we agreed with this. The value for maxGasPrice at the config file is not used.

Is possible to add to this PR in this line this suggestion and test? Or do you believe is better to create another PR to fix this @Mikelle? 

```suggestion
	maxGasPriceBig := big.NewInt(t.cfg.MaxGasPrice)
```